server/src/main/java/org/apache/iotdb/db/qp/physical/sys/CreateTimeSeriesPlan.java

```diff
@@ -133,12 +134,21 @@ public String toString() {
     return String.format("seriesPath: %s, resultDataType: %s, encoding: %s, compression: %s", path,
         dataType, encoding, compressor);
   }
-  
+
   @Override
   public List<Path> getPaths() {
     return Collections.singletonList(path);
   }
 
+  @Override
+  public List<String> getPathsStrings() {
+    List<String> ret = new ArrayList<>();
+    for (Path curPath : Collections.singletonList(path)) {
+      ret.add(curPath.getFullPath());
+    }
+    return ret;
```

Comment:
`Collections.singletonList(path.getFullPath())`

Reply:
I'll change it. Thanks for your suggestion.